src/balloon.py

```diff
@@ -43,9 +43,9 @@ def __init__(self, link, linkmirror, linkcross, ww, ee, nw, nnw, n, nne, ne, nee
         '''
```

Comment:
can we instead have a 
```suggestion
        r'''
```
and keep the slashes intact, as @maclermo suggested

Reply:
I see this is the approach recommended by [PEP 257](https://peps.python.org/pep-0257/) and made the switch.